azext_edge/edge/providers/orchestration/resources/schema_registries.py

```diff
@@ -0,0 +1,90 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+from typing import TYPE_CHECKING, Iterable, Optional
+
+from knack.log import get_logger
+from rich.prompt import Confirm
+
+from ....util.az_client import get_registry_mgmt_client, wait_for_terminal_state
+from ....util.queryable import Queryable
+
+logger = get_logger(__name__)
+
+
+if TYPE_CHECKING:
+    from ....vendor.clients.deviceregistrymgmt.operations import (
+        SchemaRegistriesOperations,
+    )
+
+
+class SchemaRegistries(Queryable):
+    def __init__(self, cmd):
+        super().__init__(cmd=cmd)
+        self.registry_mgmt_client = get_registry_mgmt_client(
+            subscription_id=self.default_subscription_id,
+        )
+        self.ops: "SchemaRegistriesOperations" = self.registry_mgmt_client.schema_registries
+
+    def create(
+        self,
+        name: str,
+        resource_group_name: str,
+        namespace: str,
+        storage_container_url: str,
+        location: Optional[str] = None,
+        description: Optional[str] = None,
+        display_name: Optional[str] = None,
+        tags: Optional[str] = None,
+        **kwargs,
+    ) -> dict:
+        # TODO: @digimaun hierarchy blob
+
+        if not location:
+            location = self.get_resource_group(name=resource_group_name)["location"]
+
+        resource = {
+            "location": location,
+            "identity": {
+                "type": "SystemAssigned",
+            },
+            "properties": {
+                "namespace": namespace,
+                "storageAccountContainerUrl": storage_container_url,
+                "description": description,
+                "displayName": display_name,
+            },
+        }
+        if tags:
+            resource["tags"] = tags
+
+        with self.console.status("Working..."):
```

Comment:
the poller should have a spinner already - or do you want to ensure there is a rich spinner onto the masses

Reply:
rich spinner is way higher quality than the az cli native one.